docs/modding/publishing.md

```diff
@@ -0,0 +1,39 @@
+# Publishing your mod
+
+## Best practices
+
+Make sure to name your mod in the form `<your name>.<mod name>`, similar to the existing default mods, like `Northstar.Client`, `Northstar.CusomServer`, ... \
+Note that the Northstar name (`Northstar.xyz`) is reserved for mods delievered with the Northstar install and should therefore **NOT** be used.
+
+It is recommended to upload the source code of your mod to a public repository like [Github](https://github.com/) to give your users a place to suggest changes and leave feedback in an organised manner.
+
+## Thunderstore
+
+The best place to publish your mod is [Thunderstore](https://northstar.thunderstore.io/). To do so, you need to package your mod as a zip with a specific folder structure. You can either set the structure up manually or use [this GitHub template](https://github.com/laundmo/northstar-mod-template)
+
+## Thunderstore package structure
+
+The Thunderstore package zip structure is as follows:
+
+```
+mods/<your name>.<mod name>/
+icon.png
+manifest.json
+README.md
+```
+
+- `icon.png`: 256x256px icon for your mod.
+- `README.md`: the description page for your mod
+- `manifest.json` outlined here: https://northstar.thunderstore.io/package/create/docs/
+
+You can put multiple mods in the `mods/` folder, but only do this if neccessary.
+
+manifest.json checker: https://northstar.thunderstore.io/tools/manifest-v1-validator/
```

Comment:
```suggestion
manifest.json checker: [https://northstar.thunderstore.io/tools/manifest-v1-validator/](https://northstar.thunderstore.io/tools/manifest-v1-validator/)
```

Gitbook doesn't auto detect links sadly, so doing it like this is necessary to make a link clickable.